e2e/commands/add.e2e.js

```diff
@@ -17,10 +17,31 @@ describe('bit add command', function () {
   after(() => {
     helper.destroyEnv();
   });
+  describe('add before running "bit init"', () => {
+    it('Should return message to run "bit init"', () => {
+      let error;
+      try {
+        helper.createComponent('bar', 'foo.js');
+        helper.addComponent(path.normalize('bar/foo.js'));
+      } catch (err) {
+        error = err.message;
+      }
+      expect(error).to.include('fatal: scope not found. to create a new scope, please use `bit init');
+    });
+  });
   describe('add one component', () => {
     beforeEach(() => {
       helper.reInitLocalScope();
     });
+    it('Should tell the user there is already a scope', () => {
```

Comment:
Where is the description of that you run init twice?

Reply:
fixed